src/Repository/Tags.php

```diff
@@ -39,18 +39,22 @@ public function push(): self
         return $this;
     }
 
-    public function add(Name $name, Message $message): self
+    public function add(Name $name, Message $message = null): self
     {
-        ($this->binary)(
-            $this
-                ->binary
-                ->command()
-                ->withArgument('tag')
-                ->withShortOption('a')
-                ->withArgument((string) $name)
+        $command = $this
+            ->binary
+            ->command()
+            ->withArgument('tag')
+            ->withShortOption('a')
```

Comment:
When no message provided but the `-a` option is provided it will prompt to write a message. This option should be added only when a message is provided